includes/Modules/Analytics/Advanced_Tracking/Script_Injector.php

```diff
@@ -54,10 +54,11 @@ public function inject_event_script( $events ) {
 			return;
 		}
 
-		// Get file contents of script and add it to the page, injecting event configurations into it.
-		$filename = 'analytics-advanced-tracking.js';
-		if ( class_exists( '\Google\Site_Kit\Core\Assets\Manifest' ) && isset( Manifest::$assets['analytics-advanced-tracking'] ) ) {
-			$filename = Manifest::$assets['analytics-advanced-tracking'];
+		$filename = Manifest::get_filename( 'analytics-advanced-tracking' );
+		if ( is_null( $filename ) ) {
```

Comment:
Similarly here as well 🙂 
```suggestion
		if ( ! $filename ) {
```

Reply:
https://github.com/google/site-kit-wp/pull/4095/commits/c8d815593f99020e7988a6cd053bc87c43ee08ac